app/helpers/manual-scan-status.ts

```diff
@@ -1,20 +1,22 @@
-/* eslint-disable prettier/prettier */
 import { helper } from '@ember/component/helper';
 import ENUMS from 'irene/enums';
 
-export function manualScanStatus(params) {
+export function manualScanStatus(params: Array<number | string>) {
```

Comment:
is this helper getting `string` values ? otherwise we can change this to `[number]` and remove parseInt logic